addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-service.js

```diff
@@ -271,6 +275,89 @@ class EnvironmentScService extends Service {
     return result;
   }
 
+  async changeWorkspaceRunState(requestContext, { id, operation }) {
+    const existingEnvironment = await this.mustFind(requestContext, { id });
+
+    // Make sure the user has permissions to change the environment run state
+    await this.assertAuthorized(
+      requestContext,
+      { action: 'update-sc', conditions: [this._allowAuthorized] },
+      existingEnvironment,
+    );
```

Comment:
Is the user which is being granted authorization same as the user which provisioned the EC2 instance in the first place?

Reply:
Both the user that provisioned the instance and Admin has access to update the environment. The implementation is here -> https://github.com/awslabs/service-workbench-on-aws/blob/master/addons/addon-base-raas/packages/base-raas-services/lib/environment/environment-authz-service.js#L54